examples/http/time-server.cpp

```diff
@@ -77,7 +77,8 @@ int caf_main(caf::actor_system& sys, const config& cfg) {
   }
   // Note: the actor system will only wait for actors on default. Since we don't
   // start actors, we need to block on something else.
-  std::cout << "Server is up and running. Press <enter> to shut down.\n";
+  std::cout << "Server is up and running. Press <enter> to shut down."
+            << std::endl;
```

Comment:
Just curious: Why?

Reply:
Because `stdout` isn't line-buffered on Windows. This had the effect that the examples ran but just didn't print anything when I tried them on Windows. Here's what past me had to say about the change in [a27358d](https://github.com/actor-framework/actor-framework/pull/1412/commits/a27358d36eaf2d329503e96122e25c8b0358e211):

> On Windows, STDOUT is "fully buffered" by default. This means that
messages don't appear after a newline on screen, like they do on UNIX
systems that use line buffering by default. Using `std::endl` makes sure
that written messages actually appear on screen for Windows users as
they should..